flowgate-common/src/main/java/com/vmware/flowgate/common/model/ValueUnit.java

```diff
@@ -68,7 +68,8 @@ public void setKey(ValueType key) {
    public enum ValueType {
       TEMP, HUMIDITY, PDU_RealtimeVoltage, PDU_RealtimeVoltagePercent, PDU_RealtimePower,
       PDU_RealtimePowerPercent, PDU_RealtimeLoad, PDU_RealtimeLoadPercent,AirFlow,AirPressure,
-      ContactClosure,Smoke,Water,Vibration
+      ContactClosure,Smoke,Water,Vibration,RealtimeVoltage,RealtimeActivePower,RealtimeApparentPower,
```

Comment:
Let's disscuss how should we define the value type. ideally they should be the unit instead of metric names.